nist/system/ksp-nist-cm-11-1-allow-user-installation-of-software-only-with-explicit-privileged-status.yaml

```diff
@@ -0,0 +1,28 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-nist-cm-11-1-allow-user-installation-of-software-only-with-explicit-privileged-status
```

Comment:
please check if the control is correct. is it `cm-11-1` or `cm-11-2` @AbhinavCSY